content/en/blog/_posts/2021-07-27-volume-populators-alpha.md

```diff
@@ -0,0 +1,207 @@
+---
+title: Volume Populators Redesign
+date: 2021-07-27
```

Comment:
What's the new target publication date?

Reply:
I think it's 08/30/21, I'll update when I push my next commit